### PR TITLE
Social connect config stuff

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

File: shoebox/conf/prod/securesocial.conf [hunk 1]
```diff
@@ -28,8 +28,8 @@ securesocial {
 #    authorizationUrl="https://graph.facebook.com/oauth/authorize"
     authorizationUrl="https://www.facebook.com/dialog/oauth" 
     accessTokenUrl="https://graph.facebook.com/oauth/access_token"
-    clientId=530357056981814
-    clientSecret=cdb2939941a1147a4b88b6c8f3902745
+    clientId=104629159695560
+    clientSecret=352415703e40e9bb1b0329273fdb76a9
```
Comment (yonatanm) [inline shoebox/conf/prod/securesocial.conf:32]: Do we have new app ?

Comment (yonatanm) [inline shoebox/conf/prod/securesocial.conf:32]: Should also upfate in chat controller

Comment (eishay) [inline shoebox/conf/prod/securesocial.conf:32]: yes

Comment (eishay) [inline shoebox/conf/prod/securesocial.conf:32]: fixed

**Pull request merged**